docs/prover/cli/options.md

```diff
@@ -1444,6 +1444,33 @@ When you suspect you have an old installation. To install the newest version, us
 certoraRun --version
 ```
 
+Conf file options
+=================
+
+(--override_base_config)=
+## `--override_base_config`
+
+**What does it do?**
+Allows you to import flags from another conf file. The `--override_base_config` gets as a value a path to the imported conf file. If the path is relative, it is 
```

Comment:
We call them attributes in other parts of the documentation, not flags, see https://docs.certora.com/en/latest/docs/prover/cli/conf-file-api.html

Reply:
in options.md the current terminology is `flags`. It is true that in other places (especially when setting can be from a conf file or from CLI the conventions are looser